app/src/main/java/org/mozilla/fenix/settings/wallpaper/WallpaperSettings.kt

```diff
@@ -48,33 +56,111 @@ import org.mozilla.fenix.wallpapers.Wallpaper
+            if (wallpapers.isNotEmpty()) {
+                WallpaperGroupHeading(
+                    collection = collection,
+                    onLearnMoreClick = onLearnMoreClick,
+                )
+
+                Spacer(modifier = Modifier.height(16.dp))
+
+                WallpaperThumbnails(
+                    wallpapers = wallpapers,
+                    defaultWallpaper = defaultWallpaper,
+                    loadWallpaperResource = loadWallpaperResource,
+                    selectedWallpaper = selectedWallpaper,
+                    onSelectWallpaper = onSelectWallpaper,
+                )
+
+                Spacer(modifier = Modifier.height(32.dp))
+            }
+        }
+    }
+}
+
+@Composable
+private fun WallpaperGroupHeading(
+    collection: Wallpaper.Collection,
+    onLearnMoreClick: (String) -> Unit,
+) {
+    // Since the last new collection of wallpapers was tied directly to an MR release,
+    // it was decided that we should use string resources for these titles
+    // and descriptions so they could be localized.
+    // In the future, we may want to either use the dynamic wallpaper properties with localized fallbacks
+    // or invest in a method of localizing the remote strings themselves.
+    if (collection.name == "classic-firefox") {
+        Text(
+            text = stringResource(R.string.wallpaper_classic_title),
+            color = FirefoxTheme.colors.textSecondary,
+            style = FirefoxTheme.typography.subtitle2,
+        )
+    } else {
+        Text(
+            text = stringResource(R.string.wallpaper_limited_edition_title),
+            color = FirefoxTheme.colors.textSecondary,
+            style = FirefoxTheme.typography.subtitle2,
+        )
+        Spacer(modifier = Modifier.height(2.dp))
+
+        val annotatedString = buildAnnotatedString {
+            withStyle(
+                style = SpanStyle(
+                    color = FirefoxTheme.colors.textSecondary,
+                ),
+            ) {
+                append(stringResource(R.string.wallpaper_limited_edition_description))
+            }
+            collection.learnMoreUrl?.let { pushStringAnnotation(tag = "learn", annotation = it) }
+            withStyle(
+                style = SpanStyle(
+                    color = FirefoxTheme.colors.textSecondary,
+                    textDecoration = TextDecoration.Underline,
+                ),
+            ) {
+                collection.learnMoreUrl?.let { append(stringResource(R.string.wallpaper_learn_more)) }
+            }
+        }
+
+        ClickableText(
+            text = annotatedString,
+            style = FirefoxTheme.typography.caption,
+            onClick = { offset ->
+                annotatedString.getStringAnnotations(tag = "learn", start = offset, end = offset)
+                    .firstOrNull()?.let {
+                        collection.learnMoreUrl?.let { it1 -> onLearnMoreClick(it1) }
+                    }
+            },
```

Comment:
This isn't quite right. Can you look at https://searchfox.org/mozilla-mobile/source/fenix/app/src/main/java/org/mozilla/fenix/home/pocket/PocketStoriesComposables.kt#398 as an example?